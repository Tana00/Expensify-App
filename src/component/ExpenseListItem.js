import React from "react";
import { Link } from "react-router-dom";

const ExpenseListItem = ({ id, description, amount, createdAt }) => {
  return (
    <div>
      <h3>
        <Link to={`/edit/${id}`}>{description}</Link>
      </h3>
      <p>
        {amount} - {createdAt}
      </p>

      {/* {`Expense: ${props.description} costing:${props.amount} created:${props.createdAt}`} */}
    </div>
  );
};

export default ExpenseListItem;
