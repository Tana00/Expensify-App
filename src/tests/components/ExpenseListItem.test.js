import React from "react";
import { shallow } from "enzyme";
import ExpenseListItem from "../../component/ExpenseListItem";
import expenses from "../fixtures/expenses";

test("should render ExpenseListItem correctly", () => {
  const wrapper = shallow(<ExpenseListItem {...expenses[1]} />);
  expect(wrapper).toMatchSnapshot();
});
