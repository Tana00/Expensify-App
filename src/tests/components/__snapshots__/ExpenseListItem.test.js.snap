// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render ExpenseListItem correctly 1`] = `
<div>
  <h3>
    <Link
      to="/edit/2"
    >
      Rent
    </Link>
  </h3>
  <p>
    $1,095.00
    -
    December 28th, 1969
  </p>
</div>
`;
